Makefile

```diff
@@ -12,6 +12,19 @@ aw-server:
 aw-webui:
 	make -C ./aw-webui build
 
+install-git-hooks:
+	@printf "make pre-commit\n" > .git/hooks/pre-commit
+	@printf "make pre-push\n" > .git/hooks/pre-push
```

Comment:
Do git hooks always execute with the root project directory as the working directory?

Reply:
Yes, as per [the docs](https://git-scm.com/docs/githooks/2.24.0).